gitpg/giovanni/techinfo

```diff
@@ -1,6 +1,7 @@
 c++, c, python, vhdl
 Arduino IDE, Visual Studio, Eclipse, STM32Cube IDE, ISE design suite
 Windows 10, Ubuntu, Debian
+terminal: CMD y BASH
```

Comment:
Don't forget to use english ;) "y" -> "and"

Reply:
ok @J0c4, i am going to keep in mind for the next time, thanks for the feedback.